packages/core/addon/serializers/request.js

```diff
@@ -48,6 +50,27 @@ export default class RequestSerializer extends JSONSerializer {
       normalized[attr].forEach((fragment) => (fragment.source = normalized.dataSource))
     );
 
+    // add cid to sorts
+    const map = new Map();
+    normalized.columns?.forEach(({ type, field, parameters, cid }) => {
+      const canonicalName =
+        type === 'timeDimension'
+          ? canonicalizeMetric({ metric: 'dateTime', parameters })
+          : canonicalizeMetric({ metric: field, parameters });
```

Comment:
We should be able to update it to
```suggestion
      const canonicalName = canonicalizeMetric({ metric: field, parameters })
```